packages/vite/src/node/plugins/css.ts

```diff
@@ -739,10 +739,13 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           .map((file) => path.basename(file))
           .join('|')
           .replace(/\./g, '\\.')
+        // require and import calls might be chained by minifier using the comma operator
+        // in this case we have to keep one comma
+        // if a next require is chained or add a semicolon to terminate the chain.
         const emptyChunkRE = new RegExp(
           opts.format === 'es'
-            ? `\\bimport\\s*["'][^"']*(?:${emptyChunkFiles})["'];\n?`
-            : `\\brequire\\(\\s*["'][^"']*(?:${emptyChunkFiles})["']\\);\n?`,
+            ? `(\\b|,\\s*)import\\s*["'][^"']*(?:${emptyChunkFiles})["'](;|,)`
```

Comment:
I don't think you can chain top-level imports with commas, so this should probably be left as-is.

Reply:
:+1: